packages/cubejs-api-gateway/package.json

```diff
@@ -14,9 +14,9 @@
   "main": "dist/src/index.js",
   "typings": "dist/src/index.d.ts",
   "scripts": {
-    "test": "npm run unit",
+    "test": "yarn run unit",
```

Comment:
For this type of change, it's better to change it across the whole repository, because different variants in multiple places can produce misunderstandings.

Do you have a strong opinion that we need to use `yarn` instead of `npm`? I don't think that it's an important change, because npm is a default package manager which is installed on all systems with node.js

Reply:
No strong opinion from my side

Only issue with it, I have warning in the console, since we run yarn test from root directory and then yarn calls npm. Like:
```
❯ yarn test
yarn run v1.22.17
$ npm run unit
npm WARN lifecycle The node binary used for scripts is /var/folders/21/cwdmycgd6jl_l5ngd53h6ktc0000gn/T/yarn--1647596852752-0.47937816308405234/node but npm is using /Users/dmitrika/.nvm/versions/node/v14.19.0/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.
```

I'll revert it to avoid confusion 👍